public/javascripts/payments.js

```diff
@@ -242,23 +254,34 @@
     };
     // Disable the Pay button to prevent multiple click events.
     submitButton.disabled = true;
+    submitButton.textContent = 'Processing Payment…';
 
     // Create the order using the email and shipping information from the form.
+    // For Demo purposes we only create an order / a PaymentIntent on form submit.
+    // In a real application you should create this before entering the checkout.
     const order = await store.createOrder(
       config.currency,
       store.getOrderItems(),
       email,
-      shipping
+      shipping,
+      (payment === 'card')
```

Comment:
What do we think about putting this in a variable so it's explicit that we are using payment intents? Otherwise it's a bit cryptic without looking at the method contract. eg: 

```
const usesPaymentIntent = payment === 'card';
    const order = await store.createOrder(
      config.currency,
      store.getOrderItems(),
      email,
      shipping
      shipping,
      usesPaymentIntent
   );

    if (usesPaymentIntent) {
       // etc etc
    }
```

Reply:
Good call, thanks! :)